usage-examples/editorial-commands.ly

```diff
@@ -0,0 +1,34 @@
+\version "2.19.42"
+
+\include "scholarly/package.ly"
+
+\setOption scholarly.editorial.addition #`(
```

Comment:
Yes. This way it can become a separate module (which the annotate module will implicitly load but not the other way round).

I think "editions" is generally used to refer not to a single modification but to a whole (like score vs. part or manuscript vs. original-edition).  So some usable names might be
- editorial-functions
- editorial-commands
- editorials
- editorial
- highlight (scholarly.highlight) ???